app/views/teacher_interface/further_information_request_items/edit.html.erb

```diff
@@ -17,5 +17,26 @@
     <p class="govuk-body">You can upload this document on the next screen.</p>
   <% end %>
 
+  <% if @further_information_request_item.work_history_contact? %>
+    <% @work_history = @further_information_request_item.work_history %>
```

Comment:
```suggestion
    <% work_history = @further_information_request_item.work_history %>
```

Not a big thing, but since we're only using this locally we shouldn't define it as a global variable (which is what the `@` means).